src/appmixer/google/gmail/DownloadAttachment/component.json

```diff
@@ -32,40 +31,27 @@
                         "type": "string"
                     }
                 },
-                "required": ["messageId", "attachmentId", "fileName"]
+                "required": ["messageId", "attachmentId"]
```

Comment:
`fileName` is not reuquired, however flow throws the `missing file name param` if it's not provided 
![image](https://github.com/user-attachments/assets/94042b8e-9494-43ce-a66f-71bb9c37cc5a)